src/display/canvas.js

```diff
@@ -1197,7 +1138,7 @@ class CanvasGraphics {
     }
   }
 
-  endDrawing() {
+  endDrawing(applyHCMFilter = false) {
```

Comment:
Do we need this parameter, since all call-sites pass in `true` anyway?

Reply:
It's also used here: https://github.com/mozilla/pdf.js/blob/85166c60fd8363b428176f0e62adb3db560f7c70/src/display/pattern_helper.js#L584.

Reply:
You're obviously correct about the usage, however in practice this parameter doesn't look necessary anyway :-)

This is how we initialize the `graphics`-variable:
https://github.com/mozilla/pdf.js/blob/85166c60fd8363b428176f0e62adb3db560f7c70/src/display/pattern_helper.js#L551
which calls the following code, where `pageColors` isn't being included:
https://github.com/mozilla/pdf.js/blob/85166c60fd8363b428176f0e62adb3db560f7c70/src/display/canvas.js#L2405-L2417

So, unless I'm missing something, it seems that it should work without this new parameter!?

Reply:
Ah yes you're right.
I'm not sure it helps to make the code clear for future readers (we potentially are ;)).
Anyway, if for any reason we want to use this function somewhere else then we'll have to remind that maybe we don't want to apply the final filter, so at least the parameter we've to be explicit.

Reply:
What if we simplified things by instead directly passing in the `pageColors`-option to the `endDrawing`-method, when calling it from the API?